terminus-ui/card/src/card.component.ts

```diff
@@ -154,6 +154,22 @@ export class TsCardComponent {
   }
   public _disabled = false;
```

Comment:
This should also be isDisabled.

Reference in button component: 
https://github.com/GetTerminus/terminus-ui/blob/master/terminus-ui/button/src/button.component.ts#L207-L219

Reply:
We can't actually remove the `disabled` input yet since that would be a breaking change. Just updating internals and adding the new input so that apps can begin the upgrade. So for now, both inputs will exist.

Reply:
Ah, thank you for clarifying. Carry on Dean!